iommu_intro.adoc

```diff
@@ -583,6 +583,12 @@ and has several interfaces (see <<fig:iommu-interfaces>>):
 .. To receive "Page Request" and "Stop Marker" messages from the endpoints and
    to send "Page Request Group Response" messages to the endpoints.
 
+The interfaces related to recording an incoming MSI in a memory-resident
+interrupt file (MRIF) (See RISC-V Advanced Interrupt Architecture cite:[AIA])
+are implementation-specific. The partitioning of the responsibility between
+the IOMMU and the IO bridge for recording of the incoming MSI in an MRIF and the
+generation of the associated _notice_ MSI are implementation-specific.
```

Comment:
A few edits to the final sentence:

"The partitioning of responsibility between the IOMMU and the IO Bridge for recording the incoming MSI in an MRIF and generating the associated _notice_ MSI is implementation-specific."

Otherwise, I think this captures the relevant points. Thanks!